src/marketing/price-rules-catalog.md

```diff
@@ -10,10 +10,24 @@ For example, you can define and set the conditions for a price rule that when me
 
 - {:.ce-only}For recurring promotions in Magento Open Source, you can manually set a saved rule to Active or Inactive status each time you want to run the promotion.
 
+## Access catalog price rules
+
+1. On the _Admin_ sidebar, go to **Marketing** > _Promotions_ > **Catalog Price Rules**.
+
 ![Catalog price rules]({% link marketing/assets/price-rule-catalog-grid-ee.png %}){: .zoom}
```

Comment:
This is not indented and will break numbering

Reply:
Thank you @leslietilling .
Done